CMakeLists.txt

```diff
@@ -3,46 +3,88 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 # cmake-format: on
 
-cmake_minimum_required(VERSION 3.23)
+set(CMAKE_SKIP_TEST_ALL_DEPENDENCY FALSE)
 
-option(BUILD_TESTING "Build tests" ON)
+cmake_minimum_required(VERSION 3.25...3.31)
 
 project(
-    beman.inplace_vector
+    beman_inplace_vector
     VERSION 1.0.0
-    DESCRIPTION
-        "A dynamically-resizable vector with fixed capacity and embedded storage"
+    DESCRIPTION "A dynamically-resizable vector with fixed capacity and embedded storage"
     LANGUAGES CXX
 )
 
-include(GNUInstallDirs)
+if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
+    message(FATAL_ERROR "In-source builds are not allowed!")
+endif()
+
+option(INPLACE_VECTOR_BUILD_TESTING "Build tests" ${PROJECT_IS_TOP_LEVEL})
```

Comment:
Use `BEMAN_INPLACE_VECTOR_BUILD_TESTS` to conform with [[CMAKE.SKIP_TESTS]](https://github.com/beman-project/beman/blob/main/docs/BEMAN_STANDARD.md#cmake).

Reply:
OK